### PR TITLE
Fix docker image building + readme

### PR DESCRIPTION
**Pull request by AlexNik**
1. Add +x permission for `quarkus/container/ubi-null.sh`
2. Update `quarkus/container/README.md` to the correct command

### CODE REVIEW

**Comment by AlexNik** [comment]
@hmlnarik @agagancarczyk ?

**Comment by AlexNik** [comment]
@pedroigor @pskopek ?

**Review by pskopek — CHANGES_REQUESTED** [review]
The change of permission for the script quarkus/container/ubi-null.sh is not necessary.
The see the usage [in Dockerfile here](https://github.com/keycloak/keycloak/blob/d6f453f0d6d2946b5ea1637f3cfe89ceb999783d/operator/Dockerfile#L4) or [here](https://github.com/keycloak/keycloak/blob/86b280349b17aa401308c5a4ef21e726d795233f/quarkus/container/Dockerfile#L20).

File: quarkus/container/README.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ It is possible to download the Keycloak distribution from a URL:
 Alternatively, you need to build the local distribution first, then copy the distributions tar package in the `container` folder and point the build command to use the image:
 
     cp $KEYCLOAK_SOURCE/quarkus/dist/target/keycloak-<VERSION>.tar.gz .
-    docker build --build-arg KEYCLOAK_DIST=keycloak-<VERSION>.tar.gz . -t <YOUR_TAG>
+    docker build --build-arg KEYCLOAK_DIST=keycloak-<VERSION>.tar.gz -t <YOUR_TAG> .
```
[the remark this answers is not in the document]
Comment (AlexNik) [reply quarkus/container/README.md:13]: I've updated.

**Comment by AlexNik** [comment]
@pskopek up

**Comment by AlexNik** [comment]
@ahus1 could you review this PR please?

**Comment by ahus1** [comment]
@AlexNik - I see that @pskopek mentioned that changing the permissions on the shell script is not necessary, and that the change is still part of the PR. Please update your PR, or describe why the change in permissions is necessary.

**Comment by AlexNik** [comment]
Ok.
Indeed, with the current usage, it is possible to not give permissions +x on the file. But I think it's good manners to set +x permissions on files that have have permissions to run, rather than run them via `bash xxx.sh`. As a last resort, you could not change the permissions on the file in the repository itself, but change the Dockerfile where the copying takes place:
`COPY --chmod=+x target/ubi-null.sh /tmp/`
But I prefer to set the permissions correctly and not bother.

**Comment by AlexNik** [comment]
@ahus1?

**Comment by AlexNik** [comment]
@ahus1 @pskopek ?

**Review by ahus1 — APPROVED** [review]
Thank you for the PR to update the docs!

**Pull request merged**